include/solvers/vof.h

```diff
@@ -721,6 +643,47 @@ class VolumeOfFluid : public AuxiliaryPhysics<dim, GlobalVectorType>
   void
   reinitialize_interface_with_geometric_method();
 
+  /**
+   * @brief Helper function to reinit the face velocity with the adequate solution.
+   * This prevents code duplication throughout the tracer. The function looks at
+   * the multiphysics interface to decide if the velocity is a block velocity or
+   * a regular velocity. Furthermore, it also checks if a time-averaged solution
+   * is required. Otherwise the code here would be copied four times.
+   *
+   * @param[in] velocity_cell the iterator to the cell where the velocity is to
+   * be reinitialized.
+   *
+   * @param[in] face_no the number of the face where the velocity is to be
```

Comment:
face_no is it the number of faces or the face index? I think the brief is a bit unclear.